sds_data_manager/lambda_code/SDSCode/pipeline_lambdas/batch_starter.py

```diff
@@ -180,43 +180,52 @@ def filter_conditions(table):
             )
         return conditions
 
-    # First check to see if there are any jobs in progress and get the max version
+    # Step 1: query to get the max version from the processing jobs table
     max_version_record = (
         session.query(models.ProcessingJob)
         .filter(*filter_conditions(models.ProcessingJob))
         .order_by(models.ProcessingJob.version.desc())
         .first()
     )
     if max_version_record:
-        max_version = max_version_record.version
-        # If there is a job already in progress, determine whether the current job
-        # is a duplicate of the in-progress job by checking the dependency file hash.
-        # If the hashes are different, then we know the dependencies have changed and
-        # we should bump the version number and continue with processing.
+        max_version_proc = max_version_record.version
```

Comment:
Can you rename this? I don't know what `proc` means. I think it means processing_table_max_version, is that right?
```suggestion
        max_version_proc = max_version_record.version
```